### ISSUE TITLE
Q & A

### ISSUE THREAD

**Issue by Sameil14** [Sameil14/Problem-Description#2]

**Comment by RayAmofa** [comment]
Good evening fellow mates I have some questions for the gimpa library and a few of their processes.

1. What are the biggest challenges the library department currently faces?

2. How do you currently manage and organize the library's inventory?

3. Are there any difficulties in providing easy access to resources for library users?

4. What technology infrastructure is currently in place for research purposes?

**Comment by j-asare** [comment]
Solid question chale 

We can add how the process of managing book loans and returns are handled

**Comment by RayAmofa** [comment]
Okay I agree so how should we proceed?

**Comment by j-asare** [comment]
We have to get more questions then we ask am moro

**Comment by RayAmofa** [comment]
I agree so let's put all the questions we have here by tonight then tomorrow well compile them and send them out for responses.

**Comment by j-asare** [comment]
Solid

**Comment by kuxnation** [comment]
Sure I have some written down posting them in no time

**Comment by kuxnation** [comment]
How do you handle the check in and check out process for library materials?

**Comment by RayAmofa** [comment]
I think that's a good question because it is usually done manually so I think if we automate the process it will be best.

**Comment by j-asare** [comment]
I suggest we go for “1 & 2” by making it one question

**Comment by j-asare** [comment]
1 and 2 are in sync so it’s going to be easy to tackle

**Comment by NathanielGyamfi** [comment]
1&2 it is then

**Comment by RayAmofa** [comment]
Yes I agree so so how should we proceed?

**Comment by j-asare** [comment]
Unless anyone thinks otherwise

**Comment by NathanielGyamfi** [comment]
If no counter comment i assume everyone agree then

**Comment by j-asare** [comment]
> Yes I agree so so how should we proceed?

We will ask them the question tomorrow and knowing how the library is no one has to tell us they do most of their inventory manually 😂

**Comment by RayAmofa** [comment]
I mean what are some of the challenges the library department faces and how do they currently organize the inventory in the library.

**Comment by NathanielGyamfi** [comment]
I think we should proceed

**Comment by kuxnation** [comment]
> > Yes I agree so so how should we proceed?
> 
> We will ask them the question tomorrow and knowing how the library is no one has to tell us they do most of their inventory manually 😂

😂😂 I tell u

**Comment by kuxnation** [comment]
Then I guess we going there tomorrow right?

**Comment by kuxnation** [comment]
And I think question 4 to is a good question that we can look at

**Comment by RayAmofa** [comment]
Yes we will need to create a questionnaire about the problems that we asked them about last time.

**Comment by j-asare** [comment]
> Yes we will need to create a questionnaire about the problems that we asked them about last time.

Yeahhh true true

**Comment by kuxnation** [comment]
> Yes we will need to create a questionnaire about the problems that we asked them about last time.

Yep that will make things simple

**Comment by RayAmofa** [comment]
Great so let's discuss what we will put on the questionnaire so tomorrow well submit it to them and get our responses.

**Comment by kuxnation** [comment]
Sure

**Comment by RayAmofa** [comment]
Hello my people I think we should tackle the problem of the tracking  of inventory by the library any solution you guys have in mind before I chip in?

**Comment by Sameil14** [comment]
> Hello my people I think we should tackle the problem of the tracking of inventory by the library any solution you guys have in mind before I chip in?

yes ...That will be a good idea

**Comment by RayAmofa** [comment]
I thinking if we combine excel and the incorporate the calendar app mainly the reminder section with the system it will be much more efficient for both the librarian and the student borrowing the book giving them constant reminders of books to be returned this will help keep better track of the books.

**Comment by RayAmofa** [comment]
Hello my people good morning what do you think about the suggested solution I gave?

**Comment by Sameil14** [comment]
> Hello my people good morning what do you think about the suggested solution I gave?

I agree with you bro

**Comment by Sameil14** [comment]
> I thinking if we combine excel and the incorporate the calendar app mainly the reminder section with the system it will be much more efficient for both the librarian and the student borrowing the book giving them constant reminders of books to be returned this will help keep better track of the books.

But the question is how do we go about implementing what you're are saying ?

**Comment by NathanielGyamfi** [comment]
Should we proceed to the implementation

**Comment by RayAmofa** [comment]
Yes we should proceed to the implementation of that solution in the current GIMPA library.

**Comment by kuxnation** [comment]
👍🏼

**Comment by j-asare** [comment]
Straight up

**Comment by RayAmofa** [comment]
So how are we going to about it my people?

**Comment by kuxnation** [comment]
Any suggestions

**Comment by RayAmofa** [comment]
Hello my people

**Comment by RayAmofa** [comment]
The following is part of the implementation 
Make a spreadsheet: To enter and arrange book information such as title, author,
and availability status, construct a spreadsheet using Google Sheets or Microsoft Excel.

**Comment by Sameil14** [comment]
Hello guys ... so what's our next move now?

**Comment by NathanielGyamfi** [comment]
We research on the zuho and see how best we could integrate it into our project

**Comment by Sameil14** [comment]
Oh ok

**Comment by RayAmofa** [comment]
My people how are going to use the soho to merge the forms and the spreadsheet and the calendar?

**Comment by j-asare** [comment]
Good evening guys, so I checked on the zoho sir mentioned

**Comment by j-asare** [comment]
> Good evening guys, so I checked on the zoho sir mentioned 

And all it does is basically to sync one app to the other

**Comment by j-asare** [comment]
> > Good evening guys, so I checked on the zoho sir mentioned 
> 
> And all it does is basically to sync one app to the other 

I don't know if anyone has researched on it ?

**Comment by RayAmofa** [comment]
Yeah I did some research as well it's paid as well too or?

**Comment by j-asare** [comment]
> Yeah I did some research as well it's paid as well too or?

Yeah and even with the free trial just on trying to get used to it koraa I had limited chances

**Comment by Sameil14** [comment]
Yeah me too

**Comment by j-asare** [comment]
> Yeah I did some research as well it's paid as well too or?

I found another site we can also use it's called zapier* it's works just like the zoho too but also paid u bab

**Comment by j-asare** [comment]
> Yeah me too

So I found an alternate way to sync the Google calendar,sheets and forms all together on YouTube I'm sending the link below

https://youtu.be/55cuC-uEbuQ?si=X4Pi1nwCQjyQcXXU

**Comment by j-asare** [comment]
> > Yeah me too
> 
> So I found an alternate way to sync the Google calendar,sheets and forms all together on YouTube I'm sending the link below
> 
> https://youtu.be/55cuC-uEbuQ?si=X4Pi1nwCQjyQcXXU

So with this you don't need an external app for the linkage you can do everything on google just as the "zoho" would have done

**Comment by RayAmofa** [comment]
Okay sure great will check it out and give feedback

**Comment by j-asare** [comment]
> > > Yeah me too
> > 
> > So I found an alternate way to sync the Google calendar,sheets and forms all together on YouTube I'm sending the link below
> > 
> > https://youtu.be/55cuC-uEbuQ?si=X4Pi1nwCQjyQcXXU
> 
> So with this you don't need an external app for the linkage you can do everything on google just as the "zoho" would have

![image](https://github.com/Sameil14/Problem-Description/assets/161381039/2fe21069-8813-4ede-b680-02014ea537ed)
![image](https://github.com/Sameil14/Problem-Description/assets/161381039/c5092a23-8667-4017-b773-e17d4fab7859)
![image](https://github.com/Sameil14/Problem-Description/assets/161381039/741a1ccd-1c52-4e2d-a3fe-9c4f993b545a)
![image](https://github.com/Sameil14/Problem-Description/assets/161381039/549874c1-dc75-4e13-88ae-e1e58052ed2c)
![image](https://github.com/Sameil14/Problem-Description/assets/161381039/b13fac9b-0151-49b5-8466-f8668aa98942)
![image](https://github.com/Sameil14/Problem-Description/assets/161381039/77274dcb-4d83-4283-b843-415b59ccf050)
![image](https://github.com/Sameil14/Problem-Description/assets/161381039/4f558e39-4bb4-4e64-9ad1-b54fd07ced92)

**Comment by Sameil14** [comment]
> going

Really... that's nice.
I Will give you feedback soon.

**Comment by j-asare** [comment]
> Yeah me too

I've sent screenshots's of how it works and the outcome and I think we can use it

**Comment by j-asare** [comment]
> > going
> 
> 
> 
> Really... that's nice.
> 
> I Will give you feedback soon.




Yeah so chale make you squad go thru the video and lemme know what you think also.

**Comment by RayAmofa** [comment]
Okay does it link all the the
Three together being the forms ,calendar and the spreadsheet or?

**Comment by j-asare** [comment]
> Okay does it link all the the
> Three together being the forms ,calendar and the spreadsheet or?

Yeah actually what it does is this
So once you create the forms and share the link to let's say the student in our case it sends a response to the librarian and also fills out the spreadsheet automatically also creating the event on calendar too all in one

**Comment by j-asare** [comment]
> > Okay does it link all the the
> > Three together being the forms ,calendar and the spreadsheet or?
> 
> Yeah actually what it does is this
> So once you create the forms and share the link to let's say the student in our case it sends a response to the librarian and also fills out the spreadsheet automatically also creating the event on calendar too all in one 

That's what I was able to do from the video that's why I want you guys to also try it so we decide on what to do cus with this there's no need to pay for anything everything is free

**Comment by Sameil14** [comment]
I'm done watching the video...  I think its a perfect tool to use .

**Comment by j-asare** [comment]
> I'm done watching the video...  I think its a perfect tool to use .


Yeah I think so too but let's wait for the rest

**Comment by j-asare** [comment]
I was thinking of something too concerning the reminder. Why don't we set it to send a reminder everyday I think that way will be more efficient as compared to just a day or two before due date.

**Comment by j-asare** [comment]
> I was thinking of something too concerning the reminder. Why don't we set it to send a reminder everyday I think that way will be more efficient as compared to just a day or two before due date.

So we can program the calendar to send mails to the students everyday

**Comment by Sameil14** [comment]
Oh ok

**Comment by j-asare** [comment]
> Oh ok

But we need to list the variables o 

As in the details we will be taking from the students

**Comment by Sameil14** [comment]
By how do we go about collecting the students infor. Are we going to use Google forms for them to fill or the librarians will collect them with the Google sheet ?

**Comment by Sameil14** [comment]
We've started the implementation that , so please it will be best to answer me in the "implementation plan " side of the 3 issues.

**Comment by RayAmofa** [comment]
I agree with this approach as the information fit students we will make it easy for them by doing it multiple choice options like department and which level then we'll create space for the index number and name and title of book.

**Comment by j-asare** [comment]
> We've started the implementation that , so please it will be best to answer me in the "implementation plan " side of the 3 issues.

I don't understand you

**Comment by j-asare** [comment]
> I agree with this approach as the information fit students we will make it easy for them by doing it multiple choice options like department and which level then we'll create space for the index number and name and title of book.

Herh chale that'll be a lot of options o to put all departments together I don't know how many departments sef we get 😂😂why don't we allow them type their own departments I mean unless you or anyone here can list all departments

**Comment by RayAmofa** [comment]
Okay true when I think about it dier

**Comment by j-asare** [comment]
> Okay true when I think about it dier

Unless anyone here knows all the departments

**Comment by RayAmofa** [comment]
Or we just put the school on a there then they will write their courses like School of technology and social sciences, Business school, law school ,public school

**Comment by RayAmofa** [comment]
Hello my people
